mbbsd/register.c

```diff
@@ -1264,7 +1262,7 @@ create_regform_request()
 }
 
 static int
-register_email_input(const char *userid, char *email)
+register_email_input(const userec_t *u, char *email)
```

Comment:
Within the commit `register: Move userid hack into emaildb functions.`, the parameter type change is unwarranted.
Please move this to the commit that actually needs it, which I assume is `register: Dual read/write to verifydb.`

Reply:
Moved.